tests/test__greenness.py

```diff
@@ -4,8 +4,15 @@
 """
 import unittest
 
-from eventlet.green import urllib2, BaseHTTPServer
+from eventlet.green import BaseHTTPServer
 from eventlet import spawn, kill
+from eventlet.support import six
+
+if six.PY2:
+    from eventlet.green.urllib2 import HTTPError, urlopen
```

Comment:
If we don't have common API in green.urllib/2, then forget about it, too much worry for few lines.